jep/1/README.adoc

```diff
@@ -500,29 +570,38 @@ Informational and Process JEPs may be updated over time to reflect changes
 to development practices and other details. The precise process followed in
 these cases will depend on the nature and purpose of the JEP being updated.
 
-Replaced::  [[replaced]]
+Replaced:: [[replaced]]
 Final JEPs may eventually also be "Replaced" - superseded by a different JEP -
 rendering the original obsolete.
 This is intended for Informational JEPs, where version 2 of an API can replace version 1.
+When a JEP is marked as replaced, the <<header-superseded-by, Superseded-by>> header
+must be filled in with a link to the new JEP.
 
 ==== Other JEP Outcomes
 
 Not all JEPs will be accepted and finalized.
 
-Rejected::  [[rejected]]
-A JEP can also be "Rejected" by the reviewer. Perhaps after all is said and done it
-was not a good idea. It is still important to have a record of this fact.
-Even an Accepted draft may ultimately be Rejected
-at some point before it reaches "Final" status,
+Rejected:: [[rejected]]
+A JEP <<Reviewer>> may choose to reject a JEP.
+Perhaps after all is said and done it was not a good idea
+or perhaps a competing proposal is a better alternative.
+It is still important to have a record of this fact.
++
+Rejecting a JEP is a very strong statement.
+If the reviewer believes the JEP might eventually be accepted with sufficient modification,
+the reviewer will not reject the JEP.
+If a reviewer is confident JEP will never be accepted,
+they should inform the JEP sponsor sooner rather than later to prevent wasted effort.
```

Comment:
Style comment: "sooner" implies the opposite of "later," thus I always find "sooner rather than later" to be superfluous. We can instead say "as soon as possible" if we want. But this is just my style, I don't mind if you don't want to change it.